docs/docusaurus/docs/core/run_validations/_examples/choose_result_format.py

```diff
@@ -0,0 +1,78 @@
+## This is the set up.
+
+import great_expectations as gx
+import great_expectations.expectations as gxe
+from great_expectations.core import ValidationDefinition
+from great_expectations.core.expectation_suite import ExpectationSuite
+
+context = gx.get_context()
+
+datasource_name = "all_csv_files"
+path_to_datasource_files = "/Users/rachelreverie/PycharmProjects/pythonProject/core_sandbox/2024_05_23_notebooks/data/taxi_yellow_tripdata_samples"
```

Comment:
Where are these example python files used? Are these going to be the source of snippets later? Either way, I'd probably use a path relative to the project. I'm guessing not a lot of readers of this have a `/Users/rachelreverie/` directory on their machine 😄

Reply:
Yeah... If these were actually under test they'd be using a relative path based on the folder that gets generated in the test environment.  This is what happens when I copy code straight out of my Jupyter notebooks! :sweat_smile:

Reply:
To answer the first question: these were intended to be the scripts that were put under test, but since we're doing that in a second past for the moment they're just waiting until then.